projenrc/publish-to-adc.task.ts

```diff
@@ -16,6 +17,11 @@ async function main() {
   if (!TARGET_BUCKETS) {
     throw new Error('Require $TARGET_BUCKETS');
   }
+  const buckets = TARGET_BUCKETS.split(/\s+|,+/).filter(x => x);
+
+  const root = 'dist/standalone';
+
+  const standaloneFiles = await glob('**/*', { cwd: root });
```

Comment:
There is a risk here of accidentally publishing something we don't intend to. Normally this wouldn't be an issue, but ADC regions have such stringent policies and processes, that I can see someone paging us over this. Should we have an allowlist?